### ISSUE TITLE
Impossible to use custom YOLOv8 when loading model

### ISSUE THREAD

**Issue by nlsferrara** [ultralytics/ultralytics#19105]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

I tried to use my own train model for inference but instead of that I download and use a new yolov8n.pt

Here is the output:
Downloading https://github.com/ultralytics/assets/releases/download/v0.0.0/yolov8n.pt to 'yolov8n.pt'...
100%|██████████| 6.23M/6.23M [00:00<00:00, 42.9MB/s]

### Environment

Ultralytics YOLOv8.0.239 🚀 Python-3.10.16 torch-2.4.0+cu121 CPU (13th Gen Intel Core(TM) i7-1365U)
Setup complete ✅ (12 CPUs, 15.5 GB RAM, 299.9/1006.9 GB disk)

OS                  Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.39
Environment         Linux
Python              3.10.16
Install             git
RAM                 15.45 GB
CPU                 13th Gen Intel Core(TM) i7-1365U
CUDA                None

matplotlib          ✅ 3.8.4>=3.3.0
numpy               ✅ 1.24.3>=1.22.2
opencv-python       ✅ 4.10.0.84>=4.6.0
pillow              ✅ 10.4.0>=7.1.2
pyyaml              ✅ 6.0.2>=5.3.1
requests            ✅ 2.32.3>=2.23.0
scipy               ✅ 1.11.4>=1.4.1
torch               ✅ 2.4.0>=1.8.0
torchvision         ✅ 0.19.0>=0.9.0
tqdm                ✅ 4.67.1>=4.64.0
psutil              ✅ 6.1.1
py-cpuinfo          ✅ 9.0.0
thop                ✅ 0.1.1-2209072238>=0.1.1
pandas              ✅ 2.0.3>=1.1.4
seaborn             ✅ 0.13.2>=0.11.0
hub-sdk             ✅ 0.0.18>=0.0.2

### Minimal Reproducible Example

from ultralyticsplus import YOLO

model = YOLO("path/to/best.pt")

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @nlsferrara, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Ultralytics Documentation](https://docs.ultralytics.com/) for new and returning users. The [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples there often provide answers to many common questions.

If this is a 🐛 **Bug Report**, it seems you’re encountering an issue with loading a custom YOLOv8 model. To help us address your concern more effectively, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) similar to the one you've shared, ensuring that paths and inputs are complete for debugging. Also, ensure your `path/to/best.pt` points to a valid model file.

For **Custom Training or Usage ❓Questions**, please include additional details like dataset image examples, training logs, or any other relevant information. Make sure to review the [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) for guidance.

### Debugging Checklist

Please check the following steps to ensure your environment is set up correctly:

1. Confirm that your `best.pt` file is valid and accessible, and resides at the specified path.
2. Verify your installed `ultralytics` package is the latest version and dependencies are satisfied. Update using:
   ```bash
   pip install -U ultralytics
   ```
3. Double-check that your Python environment meets the compatibility requirements: **Python>=3.8** and **PyTorch>=1.8**.

---

### Ultralytics Environments

If you'd like to test or reproduce this behavior in another supported environment, you can try these options with pre-installed dependencies:
- **Free GPU Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Cloud Solutions**: Utilize Google Cloud with our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/) or AWS instances via the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker**: Run YOLOv8 in a Docker container with the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/). Pull the image here: <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

### Community and Support

Keep exploring and share your findings in our community:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time discussion
- [Discourse](https://community.ultralytics.com/) for deeper technical conversations
- [Subreddit](https://reddit.com/r/Ultralytics) for broader community insights

An Ultralytics engineer will review your issue shortly to provide further guidance. Thank you for your patience and for helping us improve! 😊

**Comment by Y-T-G** [comment]
Duplicate of https://github.com/ultralytics/ultralytics/issues/17011

**Comment by nlsferrara** [comment]
I don't understand. How can I resolve that ?
